src/extension/kernel.ts

```diff
@@ -691,6 +692,7 @@ export class Kernel implements Disposable {
     }
 
     TelemetryReporter.sendTelemetryEvent('cell.startExecute')
+    getEventReporter().reportExecution(cell)
```

Comment:
the reporter call is async, we should probably await it here

Reply:
Would that block execution below? My thinking was that we want LogEvent Reporting to be out of band and not block critical processing. If I was doing this in go I'd fire off a go routine. Is calling an async function and not awaiting it similar to doing `go somefunc`?